### PR TITLE
Iframe bridge changes and example

### PR DESCRIPTION
**Pull request by fractaltheory**
(brief description / PR title)

 **JIRA**: [WEB-989](https://mobify.atlassian.net/browse/WEB-989)
 **Linked PRs**: https://github.com/mobify/progressive-web-sdk/pull/388

## Changes
- Build the `content-api.js` script as part of webpack config
- Add example of manual navigation of the iframe, via redux action
- Modify `loader.js` with new case for loading within child frame

## How to test-drive this PR
- Ensure you've followed directions in https://github.com/mobify/progressive-web-sdk/pull/388
- `npm link progressive-web-sdk`
- `npm run dev`

### CODE REVIEW

File: app/content-api.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import ChildFrame from 'progressive-web-sdk/dist/iframe/child'
+let $
+
+const waitForJQuery = new Promise((resolve) => {
```
Comment (jansepar) [inline app/content-api.js]: Could this be put into a util method in the SDK?

Comment (fractaltheory) [inline app/content-api.js]: Well, it's pretty specific to the fact that Merlin's uses Require.js for module loading.

File: app/content-api.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+})
+
+const frame = new ChildFrame({
+    debug: true,
+    willFireReady: true,
+    readyCheck: waitForJQuery
+})
+
+frame
+    .registerMethod('getText', (selector) => $(selector).text())
```
Comment (fractaltheory) [inline app/content-api.js]: I've changed the waitForJQuery method to wait for jQuery `DOMContentLoaded`, so we know this text should be here. In a case where the text was dynamically added, we'd need something more specific written anyway.

**Comment by jeffkamo** [comment]
Is this PR worth keeping open? Can we close it? @jansepar @jeremywiebe

**Comment by jeffkamo** [comment]
This PR is closed, but can act as a reference for future work.

**Pull request closed without merging**